ios/FluentUI/Core/FluentUIFramework.swift

```diff
@@ -75,7 +48,7 @@ public class FluentUIFramework: NSObject {
         }
     }
 
-    @objc public static func initializeAppearance(with primaryColor: UIColor, whenContainedInInstancesOf containerTypes: [UIAppearanceContainer.Type]? = nil) {
+    @objc public static func initializeAppearance(with primaryColor: UIColor, whenContainedInInstancesOf containerTypes: [UIAppearanceContainer.Type]? = nil, fluentTheme: FluentTheme? = nil) {
```

Comment:
we should have another convenience method with the previous version of initializeAppearace() for objc callers to remain the same and work as expected.